subprojects/docs/src/samples/toolingApi/runBuild/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+    application
+}
+
+val toolingApiVersion = gradle.gradleVersion
+
+// tag::use-tooling-api[]
+repositories {
+    maven { url = uri("https://repo.gradle.org/gradle/libs-releases") }
```

Comment:
I think there is better overload extension for doings so:
`maven(url = "https://repo.gradle.org/gradle/libs-releases")`
See https://github.com/gradle/kotlin-dsl/blob/1d81b63f3a35f4122b6a49b9f57f7381b714b9a5/subprojects/provider/src/main/kotlin/org/gradle/kotlin/dsl/RepositoryHandlerExtensions.kt#L35

Reply:
Yep, the overload can be handy. But let's stick to the more general and more close to Groovy `maven { url = uri("...") }` for user manual snippets. It also has the side effect of demonstrating how to create and assign a URI in the general case.